docs/en/structure.md

```diff
@@ -23,7 +23,7 @@ For any non-trivial app, we will likely need to leverage modules. Here's an exam
 └── store
     ├── index.js          # where we assemble modules and export the store
     ├── actions.js        # root actions
-    ├── mutations.js      # root mutations
```

Comment:
I think `mutations.js` includes mutations itself but not mutation types so we should not remove it.

Also `types.js` would be better since some users want to declare constants for getters and actions, too.